MdePkg/Include/IndustryStandard/PciExpress60.h

```diff
@@ -33,6 +33,22 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #define EFI_PCIE_CAPABILITY_DEVICE_CONTROL_3_OFFSET       0x08
 #define EFI_PCIE_CAPABILITY_DEVICE_STATUS_3_OFFSET        0x0C
 
+//

```

Comment:
Since these are from the PCI Express specs, these changes could be done as an independent PR while the rest of the DOE feature continues to be discussed.

Reply:
Done!

https://github.com/tianocore/edk2/pull/10858